volatility3/framework/plugins/windows/mftscan.py

```diff
@@ -22,7 +20,21 @@ class MFTScan(interfaces.plugins.PluginInterface, timeliner.TimeLinerInterface):
 
     _required_framework_version = (2, 0, 0)
```

Comment:
```suggestion
    _required_framework_version = (2, 26, 0)
```
This needs bumping because of the extension class changes you're making.